src/core/function.js

```diff
@@ -399,15 +399,17 @@ var PDFFunction = (function PDFFunctionClosure() {
       var range = IR[2];
       var code = IR[3];
 
-      var compiled = (new PostScriptCompiler()).compile(code, domain, range);
-      if (compiled) {
+      if (!!PDFJS.isEvalSupported) {
```

Comment:
PDFJS.isEvalSupported does not exist -- we removed all globals from PDFJS. This setting needs to be passed some other way.